experimenter/experimenter/base/__init__.py

```diff
@@ -13,7 +14,7 @@ def app_version():
 
     if app_version is None:
         try:
-            with open(settings.APP_VERSION_JSON_PATH) as version_json_file:
+            with Path.open(settings.APP_VERSION_JSON_PATH) as version_json_file:
```

Comment:
Oh I see, it's becuase `settings.settings.APP_VERSION_JSON_PATH` is already a `Path`, not a string, but I didn't know you can just call `.open()` on a `Path`, that's way more ergonomic 👍

Reply:
Hrmm one hiccup about that is it makes it a much bigger hassle to mock out, since you can mock out `Path.open` and then assert it's called with the correct path, but if you call `.open()` it's called with no args and you have to mock out the path object, assert it's called with the right constructor, then inject a `.open` method and assert it's called with no args.

I dunno what do you think?

Reply:
Oh I figured out a better solution.  If I move all the paths into `settings.py` then you can mock them there, that way the test is clearly calling out which path it's mocking and it's easy to mock the call to `.open`.  That's a better solution.